plugins/oauth/girder_oauth/providers/cilogon.py

```diff
@@ -0,0 +1,94 @@
+import warnings
+
+from girder.api.rest import getApiUrl
+from girder.exceptions import RestException
+from girder.models.setting import Setting
+
+from ..settings import PluginSettings
+from .base import ProviderBase
+import requests
+
+
+class CILogon(ProviderBase):
+    _AUTH_SCOPES = ['openid', 'email', 'profile']
+    _API_USER_URL = 'https://cilogon.org/oauth2/userinfo'
+
+    @classmethod
+    def _authority(cls):
+        return 'https://cilogon.org'
```

Comment:
Given that this is just a static value, if you don't want to repeat it, let's just do:

```suggestion
    _AUTHORITY = 'https://cilogon.org'
```

(Note that it's already repeated in the above `_API_USER_URL`.